docs/python_docs/python/tutorials/getting-started/crash-course/5-datasets.md

```diff
@@ -0,0 +1,543 @@
+
+# Install MXNet 2.0
+
+
+```python
+!pip install --pre mxnet -f https://dist.mxnet.io/python
+```
+
+    Looking in links: https://dist.mxnet.io/python
+    Collecting mxnet
+      Downloading https://repo.mxnet.io/dist/python/cpu/mxnet-2.0.0b20200831-py2.py3-none-manylinux2014_x86_64.whl (51.5 MB)
+    [K     |████████████████████████████████| 51.5 MB 7.1 MB/s eta 0:00:01
+    [?25hRequirement already satisfied: numpy<2.0.0,>1.16.0 in /home/ec2-user/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages (from mxnet) (1.18.1)
+    Collecting contextvars; python_version < "3.7"
+      Downloading contextvars-2.4.tar.gz (9.6 kB)
+    Requirement already satisfied: requests<3,>=2.20.0 in /home/ec2-user/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages (from mxnet) (2.23.0)
+    Requirement already satisfied: graphviz<0.9.0,>=0.8.1 in /home/ec2-user/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages (from mxnet) (0.8.4)
+    Collecting immutables>=0.9
+      Downloading immutables-0.14-cp36-cp36m-manylinux1_x86_64.whl (98 kB)
+    [K     |████████████████████████████████| 98 kB 3.6 MB/s eta 0:00:011
+    [?25hRequirement already satisfied: idna<3,>=2.5 in /home/ec2-user/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages (from requests<3,>=2.20.0->mxnet) (2.9)
+    Requirement already satisfied: chardet<4,>=3.0.2 in /home/ec2-user/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages (from requests<3,>=2.20.0->mxnet) (3.0.4)
+    Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /home/ec2-user/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages (from requests<3,>=2.20.0->mxnet) (1.25.8)
+    Requirement already satisfied: certifi>=2017.4.17 in /home/ec2-user/anaconda3/envs/mxnet_p36/lib/python3.6/site-packages (from requests<3,>=2.20.0->mxnet) (2020.6.20)
+    Building wheels for collected packages: contextvars
+      Building wheel for contextvars (setup.py) ... [?25ldone
+    [?25h  Created wheel for contextvars: filename=contextvars-2.4-py3-none-any.whl size=7664 sha256=c4195e935f5505bed92acbaa71e886731fa08e6496d9dc8de3ee4453cf5613ec
+      Stored in directory: /home/ec2-user/.cache/pip/wheels/41/11/53/911724983aa48deb94792432e14e518447212dd6c5477d49d3
+    Successfully built contextvars
+    Installing collected packages: immutables, contextvars, mxnet
+    Successfully installed contextvars-2.4 immutables-0.14 mxnet-2.0.0b20200831
+    [33mWARNING: You are using pip version 20.0.2; however, version 20.2.2 is available.
+    You should consider upgrading via the '/home/ec2-user/anaconda3/envs/mxnet_p36/bin/python -m pip install --upgrade pip' command.[0m
```

Comment:
In the 5th chapter of the crashcoure I think we can assume that mxnet is installed